lib/views/watch_live/watch_live.dart

```diff
@@ -53,7 +46,10 @@ class _WatchLiveState extends State<WatchLive> {
   }
 }
 
-//lobby screen
+/// If the director hasn't started the livestream yet,
+/// participants are directed to this screen and can wait for it to start.
+/// 
+/// Will redirect to [ParticipantStreamScreen] once the director is ready.
 class LobbyScreen extends StatefulWidget {
```

Comment:
I prefer to avoid making comments that just have the name, since you can just read the name in the code 🙂